packages/suite/src/components/suite/CoinList/CoinList.tsx

```diff
@@ -75,9 +75,20 @@ export const CoinList = ({
                     getCoinUnavailabilityMessage(unavailableReason);
                 const tooltipString = discoveryTooltip || lockedTooltip || unavailabilityTooltip;
 
-                const coinLabel = blockchain[symbol].backends.selected
-                    ? 'TR_CUSTOM_BACKEND'
-                    : (label as TranslationKey);
+                const hasTokens = features.includes('tokens');
+                const hasStaking = features.includes('staking');
+
+                let label: TranslationKey | undefined;
```

Comment:
Lets not use mutable variables if we don't need to. This can (and should) be extracted to the function that returns the value:

`const label = getLabelForSomethingSomething({features, isTestnet, ...})`